Add Preferred location screen

bryangaledxw opened the pull request:
# Changes in this PR

* Add preferred location task, containing the alternative PDU page

## Screenshots of UI changes

![localhost_3000_applications_95a65e54-bcb4-4ded-8967-4ef4a453675d](https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/assets/94137563/13dfa5a6-7f85-4873-bd6a-626c456dab51)
![localhost_3000_applications_95a65e54-bcb4-4ded-8967-4ef4a453675d_tasks_preferred-location_pages_alternative-pdu](https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/assets/94137563/faa5ac92-9f35-4a60-86dc-be24c282064b)
![localhost_3000_applications_95a65e54-bcb4-4ded-8967-4ef4a453675d_tasks_check-your-answers_pages_review (1)](https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/assets/94137563/571c7f45-eb27-46b1-b5d1-b30c67e7ac47)


# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

<!-- Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible. You can monitor deployment failures in CircleCI
itself and application errors are found in
[Sentry](https://ministryofjustice.sentry.io/issues/?project=4504129156218880&referrer=sidebar&statsPeriod=24h).
Both events should be automatically sent to our [Slack
channel](https://mojdt.slack.com/archives/C048BJS7S2F). -->

bryangaledxw committed:
Add alternative PDU Apply page

bryangaledxw commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Add Preferred location screen](https://trello.com/c/IyC22ACE/1230-add-preferred-location-screen)

selina-dxw commented:
Can we change the title on the task list to 'Confirm placement location'?

Otherwise looks good.

bryangaledxw committed:
Add placement location pages to Apply journey

We add a placement location task within the placement considerations section, containing our new page. We can now fully integration test our new alternative PDU page

tahb reviewed (approved):
👍 

(We ask for the pop PDU at the end of the contact details section as we left that screen in)
